stable/grafana/templates/deployment.yaml

```diff
@@ -146,3 +151,11 @@ spec:
       {{- else }}
           emptyDir: {}
       {{- end -}}
+{{- if .Values.ldap.enabled }}
+        - name: ldap
+          secret:
+            secretName: {{ .Values.ldap.name | default (include "grafana.fullname" .) }}
+            items:
+              - key: ldap.toml
```

Comment:
See my comment on secret.yaml, I think is better to name it `ldap-toml`

Reply:
updated